src/Components/PresentationalComponents/Filters/PrimaryToolbarFilters/AffectingFilter.js

```diff
@@ -13,6 +13,7 @@ const affectingFilter = (apply, currentFilter = {}) => {
         });
     };
 
+    const filterItems = getAffectingFilterOptions(isEdgeParityEnabled);
```

Comment:
It might be better to filter out the unwanted items with the feature turned off here and not have to pass down the feature flag into all the functions.

Reply:
I also went this way so that I have minimal tech-debt. But there are multiple places where affecting filter values are used e.g. affecting filter, report custom filter, and chip builders. The changes were not clear and brought more tech debt. Thus, I changed the approach to use the feature flag in components so that when we get rid of the flag, we can easily remove the flags